modules/sw360/src/test/java/org/eclipse/sw360/antenna/sw360/rest/workflow/processors/SW360EnricherTest.java

```diff
@@ -46,6 +47,11 @@
     private SW360Enricher sw360Enricher;
     private SW360MetaDataReceiver connector;
 
+    private String sourceUrl = "source_url.com";
+    private String releaseTagUrl = "releaseTagUrl.com";
+    private String swhURL = "software_heritage.com";
```

Comment:
To have tests which are nearer to reality:
```suggestion
    private String swhURL = "https://archive.softwareheritage.org/browse/release/ae93ff0b4bdbd6749f75c23ad23311b512230894/";
```